pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -121,6 +127,13 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
     },
   ];
 
+  const downloads: DownloadProps[] = _getComponent?.data?.embedded?.downloads?.map((download: any) => ({
+    label: download.naam,
+    size: download.grootte,
+    type: download.type,
+    downloadLink: download.url,
+  }));
```

Comment:
I'm thinking it might be better to simply send the `download` object (from `...embedded.downloads`) to the `DownloadTemplate`.